### PR TITLE
Updated cake version on readme

### PR DESCRIPTION
**Pull request by jrbasso**
Just update the version used on composer example for the new release.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ recommend using the [app skeleton](https://github.com/cakephp/app) as
 a starting point. For existing applications you can run the following:
 
 ``` bash
-$ composer require cakephp/cakephp:"~3.2"
+$ composer require cakephp/cakephp:"~3.3"
```
Comment (jadb) [inline README.md]: Should we switch to use composer's (now default) `^3.3` notation?

Comment (markstory) [inline README.md]: Tilde is kind of nice as it will auto update to 3.4 which I don't think caret does.

Comment (htstudios) [inline README.md]: ^3.3.0 should
https://getcomposer.org/doc/articles/versions.md#caret

Not sure what ^3.0 does though.

Comment (antograssiot) [inline README.md]: using only 2 digits, the carret and tilde will be the same

Comment (markstory) [inline README.md]: Good to know @antograssiot

Comment (ravage84) [inline README.md]: A recent blog post about this topic:
https://developer.happyr.com/always-use-caret-instead-of-tilde

Personally, I try to always use the tilde...

Comment (inoas) [inline README.md]: Shouldn't the change then for releases happen for all the libraries in our `composer.json` files. Especially the behaviour for alpha releases is different.

**Pull request merged**